test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -26,7 +26,7 @@ Filter with dateCreated in subscriptions
 --SHELL-INIT--
 dbInit CB
 brokerStart CB
-accumulatorStart
+accumulatorStart --pretty-print
```

Comment:
Other commnads use a different style, eg.:

dbInit CB
brokerStart CB 0-255
etc.

So probably we should align and use something like:

accumulatorStart prettyPrint